### ISSUE TITLE
near login does not work

### ISSUE THREAD

**Issue by bowenwang1996** [nearprotocol/near-shell#216]
After doing the usual near login steps, I got this error 
```
TypeError: keys.some is not a function
    at /usr/lib/node_modules/near-shell/index.js:95:37
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```
To reproduce, install the latest near-shell and try to do near login on tatooine.

**Comment by vgrichina** [comment]
Looks to me like nearcore might have changed response format.

**Comment by bowenwang1996** [comment]
I see. I'll fix it, but I don't understand how nearlib tests didn't catch it.

**Comment by bowenwang1996** [comment]
@vgrichina I looked at the wrong line. I am not sure whether it is related to nearcore as it fails in `let near = await connect(options);`

**Comment by vgrichina** [comment]
Fixed in 0.16.0 release

**Comment by vgrichina** [comment]
@bowenwang1996 nearlib is already updated, near-shell didn't use latest nearlib
